src/content/docs/en/guides/cms/sitecore.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Sitecore Experience Manager & Astro
+description: Add content to your project using Sitecore as your CMS.
+type: cms
+stub: true
+service: Sitecore XM
+i18nReady: false
+---
+
+[Sitecore Experience Manager™ (XM)](https://www.sitecore.com/products/experience-manager) is a powerful and intuitive CMS for the simplified creation and management of experiences across channels and devices.
+
+## Getting started
+
+1. Create a Sitecore Headless website following official [documentation](https://doc.sitecore.com/xp/en/developers/sxa/103/sitecore-experience-accelerator/create-a-headless-tenant-and-site.html).
```

Comment:
```suggestion
1. [Create a Sitecore Headless website](https://doc.sitecore.com/xp/en/developers/sxa/103/sitecore-experience-accelerator/create-a-headless-tenant-and-site.html) following Sitcore's official documentation.
```
The link text should be as descriptive as possible, especially for assistive technology. This makes it clear where the link takes the reader.